dashboard/config/blocks/GamelabJr/gamelab_clickedOn.json

```diff
@@ -11,7 +11,8 @@
     "args": [
       {
         "name": "SPRITE",
-        "type": "Sprite"
+        "type": "Sprite",
+        "defer": true
```

Comment:
We could make `defer: true` the default for event/eventLoop block inputs, and you'd have to explicitly set `defer: false` to turn it off. (This PR originally deferred all such inputs, there was no explicit `defer` option, but I thought it should be configurable).

Reply:
When might you want the `defer: false` behavior?

Reply:
It's not super important, but if you have a field input (e.g. 'when {DIR} key pressed'), then deferring it would needlessly clutter the generated code.